Core/HLE/sceKernelHeap.cpp

```diff
@@ -110,18 +110,65 @@ static u32 SysMemForKernel_536AD5E1()
 	return 0;
 }
 
+static int sceKernelFreeHeapMemory(int heapId, u32 block) {
+	u32 error;
+	KernelHeap* heap = kernelObjects.Get<KernelHeap>(heapId, error);
+	if (heap) {
+		if (block == 0)
+			INFO_LOG(SCEKERNEL, "sceKernelFreeHeapMemory(%d): heapId, block", heapId, 0);
+			return 0;
```

Comment:
There are no `{}`, so this line always runs and the lines below it never run.  Also, you could use `hleLogSuccessInfoI()`.

-[Unknown]